tests/actions/ReportTest.js

```diff
@@ -42,7 +42,7 @@ describe('actions/Report', () => {
         Pusher.init({
             appKey: CONFIG.PUSHER.APP_KEY,
             cluster: CONFIG.PUSHER.CLUSTER,
-            authEndpoint: `${CONFIG.EXPENSIFY.URL_API_ROOT}api?command=AuthenticatePusher`,
+            authEndpoint: `${CONFIG.EXPENSIFY.PRODUCTION_API_ROOT}api?command=AuthenticatePusher`,
```

Comment:
Should we use `ApiUtils.getRootApi` here or is this intentionally meant to always use the PROD api? 
_(in that case the rename makes it clear)_